src/chrome/content/rules/Geekzone.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="geekzone.co.nz (partial cdn)">
+	<target host="cdn.geekzone.co.nz" />
+	<target host="images.geekzone.co.nz" />

```

Comment:
Ooops, true.

What I meant is if a domain is on both http and https (for example registration, login and private message pages) should I have rules for those or just list as a comment with something like "301 redirect for these pages)?

At the moment we just really want to enforce the target domains ...

Cheers